### ISSUE TITLE
Add FastBoot support

### ISSUE THREAD

**Issue by melv-n** [knownasilya/ember-toastr#21]
Currently it conflicts when running an app with FastBoot.

```
TypeError: Cannot set property 'options' of undefined
    at initialize (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/addon-tree-output/ember-toastr/initializers/toastr.js:6:1)
    at Object.initialize (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/bazaar/initializers/toastr.js:45:1)
    at /Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/ember-application/system/engine.js:88:1
    at Vertices.each (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/dag-map.js:188:1)
    at Vertices.walk (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/dag-map.js:117:1)
    at DAG.each (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/dag-map.js:62:1)
    at DAG.topsort (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/dag-map.js:68:1)
    at Class._runInitializer (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/ember-application/system/engine.js:110:1)
    at Class.runInitializers (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/ember-application/system/engine.js:75:1)
    at Class._bootSync (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/ember-application/system/application.js:391:1)
    at Class.boot (/Users/myUser/Projects/bazaar/tmp/broccoli_merge_trees-output_path-DFpRGlMk.tmp/assets/ember-application/system/application.js:370:1)
    at EmberApp.buildAppInstance (/Users/myUser/Projects/bazaar/node_modules/fastboot/src/ember-app.js:205:21)
    at EmberApp.visitRoute (/Users/myUser/Projects/bazaar/node_modules/fastboot/src/ember-app.js:232:17)
    at EmberApp.visit (/Users/myUser/Projects/bazaar/node_modules/fastboot/src/ember-app.js:301:17)
    at FastBoot.visit (/Users/myUser/Projects/bazaar/node_modules/fastboot/src/index.js:79:22)
    at /Users/myUser/Projects/bazaar/node_modules/fastboot-express-middleware/src/index.js:31:14
    at app.use (/Users/myUser/Projects/bazaar/node_modules/ember-cli-fastboot/index.js:299:11)
    at Layer.handle [as handle_request] (/Users/myUser/Projects/bazaar/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/myUser/Projects/bazaar/node_modules/express/lib/router/index.js:317:13)
    at /Users/myUser/Projects/bazaar/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/Users/myUser/Projects/bazaar/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/myUser/Projects/bazaar/node_modules/express/lib/router/index.js:275:10)
    at err (/Users/myUser/Projects/bazaar/node_modules/ember-cli/lib/tasks/server/middleware/broccoli-watcher/index.js:54:11)
    at watcher.then (/Users/myUser/Projects/bazaar/node_modules/broccoli-middleware/lib/watcher-middleware.js:35:7)
    at tryCatch (/Users/myUser/Projects/bazaar/node_modules/rsvp/dist/lib/rsvp/-internal.js:198:12)
    at invokeCallback (/Users/myUser/Projects/bazaar/node_modules/rsvp/dist/lib/rsvp/-internal.js:211:13)
    at /Users/myUser/Projects/bazaar/node_modules/rsvp/dist/lib/rsvp/then.js:26:14
    at flush (/Users/myUser/Projects/bazaar/node_modules/rsvp/dist/lib/rsvp/asap.js:80:5)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by knownasilya** [comment]
I'd merge a PR to support it if you are willing to write it 👍

**Comment by mapeveri** [comment]
I sent a PR to fix this issue.

**Comment by knownasilya** [comment]
Should be resolved with v1.7.2
